i18n/qqq.json

```diff
@@ -1,11 +1,18 @@
 {
 	"@metadata": {
 		"authors": [
-			"Kghbln",
-			"Shirayuki",
-			"Андрей"
+			"MWJames",
+			"Kghbln"
 		]
 	},
-	"sdata-desc": "{{desc|name=Semantic Extra Special Properties|url=https://www.mediawiki.org/wiki/Extension:Semantic_Extra_Special_Properties}}",
-	"sdata-name": "{{notranslate}}\nName of the extension as it should be displayed."
+	"semantic-approvedrevs-desc": "{{desc|name=Semantic Approved Revs|url=hhttps://www.mediawiki.org/wiki/Extension:Semantic_Approved_Revs}}",
+	"semantic-approvedrevs-name": "{{notranslate}}\nSemantic Approved Revs"
+	"semantic-approvedrevs-property-approved-rev": "Approved revision",
+	"semantic-approvedrevs-property-approved-rev-desc": "\"$1\" identifies the revision ID of a page that has been approved and is provided by the [https://www.semantic-mediawiki.org/wiki/Extension:Semantic_Approved_Revs Semantic Approved Revs] extension.",
```

Comment:
This is identical to the content of "en.json". The content of "en.json" is shown to the translator. The content of "qqq.json" is to provide context for better translation results. Add something like `"This message describes a special property.\n\nParameter:\n* $1 holds the name of the special property" ``You can use SESP qqq file as an example here. Same for the whole of the remaining system messages.